frontend/playwright.config.ts

```diff
@@ -1,11 +1,108 @@
 import type { PlaywrightTestConfig } from '@playwright/test';
+import { defineConfig, devices } from '@playwright/test';
+
+import * as testEnv from './tests/envVars';
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
 
 const config: PlaywrightTestConfig = {
+  testDir: './tests',
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: process.env.CI ? 1 : 0,
+  /* Opt out of parallel tests on CI. */
+  workers: process.env.CI ? 1 : undefined,
+  outputDir: 'test-results',
+  /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+  reporter: process.env.CI
+  ? [['github'], ['list']]
+  // Putting the HTML report in a subdirectory of the main output directory results in a warning log
+  // stating that it will "lead to artifact loss" but the warning in this case is not accurate
+  : [['html', { outputFolder: 'test-results/_html-report', open: 'never' }]],
+  /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+  use: {
+    /* Base URL to use in actions like `await page.goto('/')`. */
+    baseURL: testEnv.serverBaseUrl,
+
+    /* Local storage to be populated for every test */
+    storageState:
+    {
+      cookies: [], // TODO: Port login procss from PageTest.cs (but use apiContext as it's faster)
```

Comment:
Is this still a TODO?

Reply:
Nope. The `loginAs` function in authHelpers.ts is that login process, ported over. I'll remove the TODO comment.

Reply:
Fixed in commit 3c01d5d8dc56ffb161fc2fad88d69f359b5e3d37.